docs/source/index.rst

```diff
@@ -0,0 +1,21 @@
+Cookiecutter Data Science
+=========================
+A logical, flexible, and reasonably standardized project structure for doing and sharing data science work.
+
+Humble Opinion
+--------------
+The default project structure reflects certain opinions about how to do collaborative data science work. These opinions grew out of our own experiences with what works and what doesn't. Some of these opinions are about workflows, and others are about tools that can make the process easier.
```

Comment:
I would remove this, and put the "Getting started" information on this page directly.